README.md

```diff
@@ -28,32 +28,26 @@ exclude it and manually include all the other Solidus componenents.
 
 You need to replace:
 
-```
+```ruby
 gem 'solidus'
 ```
 
 with:
 
-```
+```ruby
 gem 'solidus_core'
 gem 'solidus_api'
 gem 'solidus_backend'
 gem 'solidus_sample'
-gem "solidus_starter_frontend"
 ```
 
-If Solidus was already installed, you have to run:
-
-```
-bundle
-bundle exec rails g solidus_starter_frontend:install
-```
+Install our binary in the system: `gem install solidus_starter_frontend`
 
-and change all `Spree::Frontend::Config` in `SolidusStarterFrontend::Config`.
+Execute our generator that will copy our component files in your project:
+`solidus_starter_frontend`
 
-If it's the first Solidus installation you need to run the Solidus install
-before `bundle exec rails g solidus_starter_frontend:install`.
-Please, take a look at the [Solidus README](https://github.com/solidusio/solidus#installation-options).
+If Solidus was already installed with solidus_frontend you will have to change
+all `Spree::Frontend::Config` in `SolidusStarterFrontend::Config`.
 
 ## Customization
```

Comment:
Is this section still required? 🤔

Reply:
That section is referring to switching from `solidus_frontend` to our component.
I would keep it if we still consider this option feasable.

Reply:
Ahh... you mean the Customization section.
Sure, I'm going to remove it 👍

Reply:
New commit added